README.md

```diff
@@ -41,7 +41,7 @@ For example, for `demo/podinfo`:
 
 - run:
     ```console
-    kubectl port-forward --namespace=demo podinfo-xxxxxxxxxx-xxxxx 9898
+    kubectl -n demo port-forward svc/podinfo 9898:9898
```

Comment:
I think it's better for documentation to use the long version of flags (even if they are well known) so a new kubernetes user doesn't have to look it up before running the command to understand what it is doing.

```suggestion
    kubectl --namespace demo port-forward service/podinfo 9898:9898
```

What do you think?

Reply:
I agree, more verbose, but more explicit for new users. 👍